.changeset/sixty-eggs-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/cdk": patch
```

Comment:
Argh, this should have been minor, as it's adding new behaviour rather than fixing existing. This change is to the experimental pattern, so hopefully this doesn't cause too much confusion!